app/src/main/java/com/x8bit/bitwarden/data/auth/repository/AuthRepository.kt

```diff
@@ -362,8 +362,10 @@ interface AuthRepository : AuthenticatorProvider, AuthRequestManager {
 
     /**
      * Get the password strength for the given [email] and [password] combo.
+     * If no value is passed for the [email] will use the active email of the current active
+     * account via the [userStateFlow].
      */
-    suspend fun getPasswordStrength(email: String, password: String): PasswordStrengthResult
+    suspend fun getPasswordStrength(email: String? = null, password: String): PasswordStrengthResult
```

Comment:
I was going to suggest holding `email` in state like, [ExportVaultViewModel](https://github.com/bitwarden/android/blob/main/app/src/main/java/com/x8bit/bitwarden/ui/platform/feature/settings/exportvault/ExportVaultViewModel.kt#L57) does, but this works too.

Can we update usages of `getPasswordStrength` where the active email is being used implicitly to pass null now?